src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5173,6 +5175,89 @@ namespace HeatBalanceManager {
+                    ErrorsFound = true;
+                } else {
+                    if (state.dataSurface->Surface(SurfNum).Class == DataSurfaces::SurfaceClass::Window &&
+                        state.dataSurface->Surface(SurfNum).ExtBoundCond == DataSurfaces::ExternalEnvironment) {
+                        int ConstrNum = state.dataSurface->Surface(SurfNum).Construction;
+                        int MaterNum =
+                            state.dataConstruction->Construct(ConstrNum).LayerPoint(state.dataConstruction->Construct(ConstrNum).TotLayers);
+                        bool withNoncompatibleShades =
+                            (state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::Shade ||
+                             state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::WindowBlind ||
+                             state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::Screen ||
+                             state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::GlassEquivalentLayer ||
+                             state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::GapEquivalentLayer ||
+                             state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::ShadeEquivalentLayer ||
+                             state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::DrapeEquivalentLayer ||
+                             state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::ScreenEquivalentLayer ||
+                             state.dataMaterial->Material(MaterNum).Group == DataHeatBalance::MaterialGroup::BlindEquivalentLayer ||
+                             state.dataSurface->Surface(SurfNum).HasShadeControl);
+                        if (withNoncompatibleShades) {
+                            ShowSevereError(state,
+                                            "Non-compatible shades defined alongside SurfaceProperty:IncidentSolarMultiplier for the same window");
+                            ErrorsFound = true;
+                        } else {
+                            state.dataSurface->Surface(SurfNum).hasIncSolMultiplier = true;
+                            state.dataSurface->SurfIncSolMultiplier(SurfNum).Name = state.dataIPShortCut->cAlphaArgs(1);
+                            state.dataSurface->SurfIncSolMultiplier(SurfNum).SurfaceIdx = SurfNum;
+                            state.dataSurface->SurfIncSolMultiplier(SurfNum).Scaler = state.dataIPShortCut->rNumericArgs(1);
+                            state.dataSurface->SurfIncSolMultiplier(SurfNum).SchedPtr = GetScheduleIndex(state, state.dataIPShortCut->cAlphaArgs(2));
+                        }
+                    } else { // not defined for exterior window
```

Comment:
This is a confusing error message (and can also be done with `continue;` and un-indent before you test for material groups.